code_modularization/00_title.md

```diff
@@ -239,43 +310,42 @@ def fahrenheit_to_celsius(temp_f):
 Nested code is a prime target for modularization:
 
 ```python=
-def checkTemperature(degrees):
-    if degrees < 0:
-        if degrees < -273:
-            if degrees < -459:
-                print("This temperature is impossible.")
-            else:
-                print("This temperature is likely Fahrenheit.")
-        else:
-            print("This temperature is either Celsius or Fahrenheit.")
-    else:
-        print("This temperature is in Kelvin, Celsius, or Fahrhenheit.")
+def sum_even_numbers(bottom, top):
```

Comment:
Good examples, but why not just keeping checkTemperature()?

Reply:
I didn't like the old example. The module that was extracted was basically just an if statement.
I couldn't think of another example that fell within the temperature conversion example set.

Reply:
Thanks, I am happy now. Slides look great!